build-system/tasks/pr-check.js

```diff
@@ -85,6 +85,10 @@ async function prCheck(cb) {
     runCheck('gulp check-owners');
   }
 
+  if (buildTargets.has('RENOVATE_CONFIG')) {
+    runCheck('gulp check-renovate-config');
+  }
```

Comment:
`tasks/pr-check.js` contains the local `gulp pr-check` task. To run this during Travis PR builds, you'll also need to call `gulp check-renovate-config` here:

https://github.com/ampproject/amphtml/blob/bbb63ca265abdb530a7865bbed10c5571b6b7800/build-system/pr-check/checks.js#L95-L98

Also, add a call here, so the config is checked during push builds (unlike `OWNERS` checks, which runs only for PR builds).

https://github.com/ampproject/amphtml/blob/bbb63ca265abdb530a7865bbed10c5571b6b7800/build-system/pr-check/checks.js#L47-L62

Reply:
Done